example/lib/src/file_picker_demo.dart

```diff
@@ -354,6 +480,13 @@ class _FilePickerDemoState extends State<FilePickerDemo> {
                           icon: const Icon(Icons.folder),
                         ),
                       ),
+                      SizedBox(
+                        width: 250,
+                        child: FloatingActionButton.extended(
+                            onPressed: () => _pickFileAndDirectoryPaths(),
+                            label: Text('Pick files and directories'),
+                            icon: const Icon(Icons.folder_open)),
```

Comment:
Missing comma resulting in weird formatting indent.

```suggestion
                            icon: const Icon(Icons.folder_open),),
```